truss/getstarted/getstarted.go

```diff
@@ -0,0 +1,116 @@
+package getstarted
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"text/template"
+
+	log "github.com/Sirupsen/logrus"
+	gogen "github.com/golang/protobuf/protoc-gen-go/generator"
+	"github.com/pkg/errors"
+)
+
+type protoInfo struct {
+	alias string
+}
+
+func (p protoInfo) FileName() string {
+	return p.PackageName() + ".proto"
+}
+
+func (p protoInfo) PackageName() string {
+	a := p.alias
+	a = strings.Replace(a, "-", "", -1)
+	a = strings.Replace(a, "_", "", -1)
+	a = strings.Replace(a, " ", "", -1)
+
+	a = strings.ToLower(a)
+	return a
+}
+
+func (p protoInfo) ServiceName() string {
+	a := p.alias
+	a = strings.Replace(a, "-", "_", -1)
+	a = strings.Replace(a, " ", "_", -1)
+	return gogen.CamelCase(a)
+}
+
+// Do writes a default protobuf file to the current directory, in a file named
+// "default.proto". If the file exists, it prints a warning and returns a
+// non-zero exit code. The non-zero exit code is to enable using the return
+// from this function in os.Exit().
+func Do(pkg string) int {
+	const fallbackFName = "get_started"
+	if pkg == "" {
+		pkg = fallbackFName
+	}
+	pkg = removeDotProto(pkg)
+	pinfo := protoInfo{
+		alias: pkg,
+	}
+	// For convenience sake, render the help messages now. Saves us from doing
+	// nested error checks later, making the code a bit clearer.
+	nextStep, err := renderTemplate("nextStepMsg", nextStepMsg, pinfo)
```

Comment:
Possibly move rendering `nextStep` to just before you use it. Kind of out of place to have it here from my perspective.